tests/ttnn/unit_tests/test_manip.py

```diff
@@ -0,0 +1,352 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
this file should be broken down into test_creation.py, test_reduction.py and test_losses.py. And the files should be under tests/ttnn/unit_tests/operations